Uni_life_tracker/app/src/main/res/layout/activity_add_expenses.xml

```diff
@@ -0,0 +1,146 @@
+    <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/e_add_amount"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:errorEnabled="true">
+    </com.google.android.material.textfield.TextInputLayout>
+
+
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="240dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/dollar_sign"
+            android:inputType="text"
+            app:layout_constraintHorizontal_bias="0.187" app:layout_constraintVertical_bias="0.583"/>
+    <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/e_add_date"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:errorEnabled="true">
+    </com.google.android.material.textfield.TextInputLayout>
+
+
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="240dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/add_date"
+            android:inputType="text"
+            app:layout_constraintHorizontal_bias="0.187" app:layout_constraintVertical_bias="0.438"/>
+    <ImageButton
+            android:id="@+id/arrow_button"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="50dp"
+            android:layout_height="58dp"
+            android:src="@drawable/left_button_background"
+            android:padding="2dp"
+            android:scaleType="fitCenter"
+            android:background="#0000ffff"
+            app:layout_constraintHorizontal_bias="0.049"
+            app:layout_constraintVertical_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent"
+            android:layout_marginStart="16dp"
+            android:contentDescription="@string/left_arrow"/>
+
```

Comment:
I know it's not part of our original Figma UI but I think it would be good to add a submit button here so the user could save the new expense

Reply:
Will do!